conda/datreant.core/meta.yaml

```diff
@@ -1,25 +1,25 @@
 package:
   name: datreant.core
-  version: "0.6.0"
+  version: "0.7.0"
 
 source:
   # path: ../../../
   git_url: https://github.com/datreant/datreant.core.git
   # git_branch: master
-  git_tag: release-0.6.0
+  git_tag: release-0.7.0
 
 build:
-  noarch_python: True
-  script: $PYTHON setup.py install
-  number: 3
+  # noarch_python: True
+  script: $PYTHON setup.py install --old-and-unmanageable
```

Comment:
yes it keeps setuptools from creating the eggs. The eggs are suppossed to be used when packages are managed with pip. When other managers are used the normal files are preferred instead of egg files. At least this is what I can scrap together from various blog posts and documentation. Python packaging with setuptools is a good bit messy.